### ISSUE TITLE
Can't extract Pink Slime from Mob Slaughter Factory

### ISSUE THREAD

**Issue by Archangel087** [Darkosto/SkyFactory4Issues#591]
The mob slaughter factory produces pink slime which cannot be extracted via bucket, fluiduct, or fluid push addon.

Was this intentional?

I have only unlocked bonsai trees so far through prestige but I didn't see any prestige items that looked like they'd unlock extra functionality.

**Comment by Slyklaw** [comment]
Right-click a bucket on the pink tank. or configure the sides to allow pink slime and not liquid meat to use fluiducts.

**Comment by Archangel087** [comment]
I did configure it and that didn't work.

**Comment by Centac** [comment]
The mob slaughter factory is a bit broken. You can't click on the outside with a bucket, and if you go into the gui and put the bucket into the slot (which should work) it won't do anything. What you need to do is right click to open the gui, pick up the bucket with the left mouse button, and right click the bar with pink slime while still holding onto the bucket. Yeah... weird. I don't have a clue why fluiducts aren't working for you. They seemed to work when I tried them.

**Comment by Archangel087** [comment]
That worked, thank you for your help.

**Comment by Centac** [comment]
Np. Glad I could help.

**Comment by Barberfluids** [comment]
I have tried setting up singularity tanks and they don't work.  Right click on the pink storage with a bucket manually is the only way I could fill my tanks manually.

**Comment by Barberfluids** [comment]
I did get a little bit of production to start on both the liquid meat and pink slime and then it stopped.  When I positioned the unit the front was facing forward.  Does anyone know if orientation (Left, Right, Front and Back, Down and Up) based on the front of the unit or the back of the unit?  I am a Noob with Mods but have years of experience with Vanilla minecraft on PS4.

**Comment by Barberfluids** [comment]
Solution: You can run an importer with cable from the Mob Slaughter Factory to a Fluid Grid.  Right click a bucket on meat and pink slime and add them to the white list of the importer and select Fluids.  Change the tanks in the Factory to output to the side where the import comes into the factory and both fluids will get imported into your fluid grid.

**Comment by KeepOnDigging** [comment]
Another work around. Install a fluid transfer push addon.
Push both meat and slime into a tank next to the slaughter factory.
Then pipe out of the tank.
